aten/src/ATen/native/TensorIterator.cpp

```diff
@@ -38,7 +38,7 @@ void TensorIterator::reorder_dimensions() {
       }
       int64_t stride0 = operands_[arg].stride_bytes[dim0];
       int64_t stride1 = operands_[arg].stride_bytes[dim1];
-      if (is_reduction() && operands_[arg].is_output) {
```

Comment:
Is it weird that is_reduction_ isn't queried from config?

Reply:
Since the reduction parameter is copied in from config, it doesn't really matter